kms-message/CMakeLists.txt

```
@@ -260,16 +261,14 @@ if(NOT mongoc-1.0_FOUND)
 elseif(DISABLE_NATIVE_CRYPTO)
    message ("test_kms_azure_online target disabled: Not building with native crypto.")
 elseif(ENABLE_ONLINE_TESTS)
-   add_executable(test_kms_azure_online test/test_kms_azure_online.c test/test_kms.c test/test_kms_online_util.c)
-   target_include_directories(test_kms_azure_online PRIVATE ${PROJECT_SOURCE_DIR}/src)
-   target_include_directories(test_kms_azure_online PRIVATE ${PROJECT_SOURCE_DIR}/src/kms_message)
+   add_executable(test_kms_azure_online test/test_kms_azure_online.c test/test_kms_util.c test/test_kms_online_util.c)
+   target_include_directories(test_kms_azure_online PRIVATE ${PROJECT_SOURCE_DIR})
    target_compile_definitions(test_kms_azure_online PRIVATE ${KMS_MESSAGE_DEFINITIONS})
    target_link_libraries(test_kms_azure_online mongo::mongoc_shared)
    target_link_libraries(test_kms_azure_online kms_message_static)
 
-   add_executable(test_kms_gcp_online test/test_kms_gcp_online.c test/test_kms.c test/test_kms_online_util.c)
-   target_include_directories(test_kms_gcp_online PRIVATE ${PROJECT_SOURCE_DIR}/src)
-   target_include_directories(test_kms_gcp_online PRIVATE ${PROJECT_SOURCE_DIR}/src/kms_message)
+   add_executable(test_kms_gcp_online test/test_kms_gcp_online.c test/test_kms_util.c test/test_kms_online_util.c)
+   target_include_directories(test_kms_gcp_online PRIVATE ${PROJECT_SOURCE_DIR})
```

I have (strong) opinions that the `<root>/src` directory should be the root of the include-search path for all libraries.

```suggestion
   target_include_directories(test_kms_gcp_online PRIVATE ${PROJECT_SOURCE_DIR}/src)
```

I do strongly agree with the removal of `src/kms_message` as a search dir, though.

---

Done, on this and all kms-message test targets.